addons/l10n_ar_withholding/wizards/account_payment_register_views.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_account_payment_register_form" model="ir.ui.view">
+        <field name="name">account.payment.register.form</field>
+        <field name="model">account.payment.register</field>
+        <field name="inherit_id" ref="l10n_latam_check.view_account_payment_register_form"/>
+        <field name="arch" type="xml">
+            <group>
+                <group name="withholdings" colspan="2" 
+                    invisible="country_code != 'AR'">
+                    <field name="withholding_ids" invisible="not can_edit_wizard or (can_group_payments and not group_payment)">
+                        <tree editable="bottom">
+                            <field name="withholding_sequence_id" column_invisible="True"/>
+                            <field name="tax_id" domain="[('l10n_ar_withholding_payment_type', '=', parent.partner_type), ('company_id', '=', parent.company_id)]" options="{'no_open': True, 'no_create': True}"/>
+                            <field name="name" readonly="withholding_sequence_id"/>
+                            <field name="base_amount"/>
+                            <field name="amount"/>
+                            <field name="currency_id" invisible="True"/>
+                        </tree>
+                    </field>
+                    <group invisible="not can_edit_wizard or (can_group_payments and not group_payment)">
+                        <label for="net_amount" string="Net Amount" invisible="l10n_latam_check_id"/>
+                        <label for="net_amount" string="Check amount" invisible="not l10n_latam_check_id"/>
+                        <field name="net_amount" nolabel="1" invisible="not can_edit_wizard or (can_group_payments and not group_payment)"/>
+                    </group>
+                    <div colspan="2" class="o_row"
+                        invisible="(can_edit_wizard and not can_group_payments) or (can_group_payments and group_payment)">
+                    <p class="alert alert-warning" role="alert">You can't register withholdings when paying invoices of different partners or same partner without grouping</p>
```

Comment:
Functionally testing multiple invoices in different currencies, the withholdings won't be added to those through the wizard even if you group, so somehow we need to avoid it or say that we do not compute.

Reply:
1. add message that withholdings will not be applied
2. check that withholdings are not being show